packages/gatsby-source-kontent/src/template.items.schema.gql

```diff
@@ -110,6 +111,7 @@ type __KONTENT_ELEMENT_RICH_TEXT_IMAGE_VALUE__ {
   description: String
   height: Int!
   width: Int!
+  type: String!

```

Comment:
https://github.com/Kentico/kontent-gatsby-packages/issues/177#issuecomment-801948216

This should be removed (my bad)

```suggestion
```